internal/cmd/flags.go

```diff
@@ -114,6 +115,10 @@ func (f *flags) bindMethod(flagSet *pflag.FlagSet) {
 	flagSet.StringVar(&f.method, "method", "", "The GRPC method to call in the form package.Service/Method. This is required.")
 }
 
+func (f *flags) bindOutput(flagSet *pflag.FlagSet) {
+	flagSet.StringVarP(&f.outputFormat, "output", "", "", "Output format: JSON or Tabs")
```

Comment:
If we're doing to do this, this should be `--json` or similar, or explicitly say what the values are here.

Reply:
Sure, let me know what API(s) you'd prefer, and I'd be happy to make the change. I don't know, for example, whether prototool intends to implement output formats as a general feature in the future.

Reply:
Hey @mickeyreiss, thanks for this! Please use a `--json` flag similar to https://github.com/uber/prototool/pull/212. 

Otherwise, this approach looks good!

Reply:
This should be addressed by a5b04fb and 9d809df.